docs/docs/stack_config.md

```diff
@@ -137,11 +137,14 @@ Stack config supports templating in the same way Environment config can be, as d
 
 Stack config makes environment config available to template.
 
+
 ### Environment config
 
+Config item values can be replaced with parts of the environment config:
```

Comment:
What about something like:

A stack configuration file can reference a stack environment parameter using the syntax...?